package.json

```diff
@@ -54,7 +54,10 @@
     ],
     "moduleNameMapper": {
       "\\.(css|less|svg)$": "identity-obj-proxy"
-    }
+    },
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
```

Comment:
What does this `enzyme-to-json` package exactly ?

I write the following:
```
import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme';
import { IconPicker } from 'index';

describe('When using snapshots', () => {
  it('Should render with an element children', () => {
    const wrapper = shallow(<IconPicker />);

    expect(wrapper).toMatchSnapshot();
  });
});
```

without any extra package/conf, and it is ok.

Reply:
It gives a better output in the snapshot